homeassistant/components/ecowitt/__init__.py

```diff
@@ -0,0 +1,178 @@
+"""The Ecowitt Weather Station Component."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+import logging
+import time
+
+from pyecowitt import WINDCHILL_HYBRID, WINDCHILL_NEW, WINDCHILL_OLD, EcoWittListener
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_PORT,
+    CONF_UNIT_SYSTEM_IMPERIAL,
+    CONF_UNIT_SYSTEM_METRIC,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.dispatcher import (
+    async_dispatcher_connect,
+    async_dispatcher_send,
+)
+from homeassistant.helpers.entity import DeviceInfo, Entity
+
+from .const import (
+    CONF_UNIT_BARO,
+    CONF_UNIT_LIGHTNING,
+    CONF_UNIT_RAIN,
+    CONF_UNIT_WIND,
+    CONF_UNIT_WINDCHILL,
+    DATA_MODEL,
+    DATA_PASSKEY,
+    DATA_STATIONTYPE,
+    DOMAIN,
+    ECOWITT_PLATFORMS,
```

Comment:
No need to set the `ECOWITT_` prefix:
```suggestion
    PLATFORMS,
```